src/components/VoiceOutputPanel.tsx

```
@@ -123,8 +168,54 @@ export default function VoiceOutputPanel({ active }: { active: boolean }) {
     }
   }, [clipUrl]);
 
-  const post = useCallback(async (body: Record<string, unknown>) => {
+  // A sample on the box pays a full Kokoro cold start whenever kokoro-server
+  // is off — 13-17 s measured on an Orin Nano — and "Speaking…" alone for
+  // that long reads as a hang. Count the seconds, so the wait is visibly
+  // progressing, and name the reason once it is clearly a cold start.
+  useEffect(() => {
+    if (busy !== "play") return;
+    setSpeakingFor(0);
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/src/components/VoiceOutputPanel.tsx:177:5
  175 |   useEffect(() => {
  176 |     if (busy !== "play") return;
> 177 |     setSpeakingFor(0);
      |     ^^^^^^^^^^^^^^ Avoid calling setState() directly within an effect
  178 |     const startedAt = Date.now();
  179 |     const timer = window.setInterval(() => setSpeakingFor(Math.floor((Date.now() - startedAt) / 1000)), 1000);
  180 |     return () => window.clearInterval(timer);

(react-hooks/set-state-in-effect)